CHANGELOG.md

```diff
@@ -1,5 +1,80 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.8.4
+
+<!-- generated comparing v5.8.3..master -->
+
+_Jun 13, 2022_
+
+A big thanks to the 22 contributors who made this release possible.
+This release is mostly about 🐛 bug fixes and 📚 documentation improvements.
+
+### `@mui/material@5.8.4`
+
+- &#8203;<!-- 37 -->[Autocomplete] Add `getOptionKey` prop (#32910) @pushys
+- &#8203;<!-- 36 -->[Button] Add missing classes in `ButtonClasses` type (#33040) @ZeeshanTamboli
+- &#8203;<!-- 20 -->[Grid] Fix prop-type key regression (#33123) @oliviertassinari
+- &#8203;<!-- 19 -->[Grid] support customized breakpoints (#31998) @boutahlilsoufiane
```

Comment:
This one seems to be the highlight for this release.

Reply:
Yeah, makes sense 👍